graf2d/gpad/src/TPad.cxx

```diff
@@ -5031,17 +5042,27 @@ void TPad::Print(const char *filename, Option_t *option)
          }
       }
 
+      Bool_t standalone = kFALSE;
+      if (strstr(opt,"Standalone")) standalone = kTRUE;
+
       // Create a new TeX file
       if (gVirtualPS) {
          gVirtualPS->SetName(psname);
+         if (standalone) gVirtualPS->SetTitle("Standalone");
```

Comment:
Should one reset title back if "standalone" no longer specified?

Reply:
Indeed the `fTitle` variable is not used by the TTextDump class ... but it is there because it inherits from TNamed via TVirtualPS. I thought there was not need to add a new variable as there was already one not used.

Reply:
There is no need to reset because a new instance of TTexDump is created each time a canvas is printed

Reply:
```
{
  TCanvas  *canvas = new TCanvas("canvas","Canvas");
  TH1F *hist = new TH1F("hist","Title nb 4;#mbox{Energy Difference }(#Delta#mbox{E});Counts",100,-5,5);
  hist->FillRandom("gaus");
  hist->Draw();
  canvas->Print("output2.tex","Standalone");
  canvas->Print("output1.tex");
}
```
gives:
```
root [0] 
Processing latexfigure.C...
Info in <TCanvas::Print>: Standalone TeX file output2.tex has been created
Info in <TCanvas::Print>: TeX file output1.tex has been created
```
As you can see the 2nd print does not remember the "Standalone" option of the first one.